File: DAY 4/Condition.java
import java.util.Scanner;
public class Condition {
    public static void main(String[] args){
        int N,ans=1;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter N:");
        N=sc.nextInt();
        int k=N;
        while(k>0){
            System.out.print(k+" ");
            k--;
        }
        System.out.println("Blastoff!");
        for(int i=1;i<=N;i++){
            ans*=i;
        }
        System.out.println("The Factorial of "+N+" is "+ans);
        sc.close();
    }
}

File: DAY 4/Condition.cpp
#include<iostream>
using namespace std;
int main(){
    int N,ans=1;
    cout<<"Enter N:";
    cin>>N;
    int k=N;
    while(k>0){
        cout<<k<<" ";
        k--;
    }
    cout<<" Blastoff\n";
    for(int i=1;i<=N;i++){
        ans*=i;
    }
    cout<<"The Factorial is:"<<ans;
    return 0;
}